app/api/attendees.py

```diff
@@ -25,9 +26,23 @@
 from app.models.ticket_holder import TicketHolder
 from app.models.user import User
 
+from app.settings import get_settings
+
 attendee_misc_routes = Blueprint('attendee_misc', __name__, url_prefix='/v1')
 
 
+def get_sold_and_reserved_tickets_count(event_id):
+    order_expiry_time = get_settings()['order_expiry_time']
+    return db.session.query(TicketHolder.id).join(Order).filter(TicketHolder.order_id == Order.id) \
+        .filter(Order.event_id == int(event_id),
+                Order.deleted_at.is_(None),
+                or_(Order.status == 'placed',
+                    Order.status == 'completed',
+                    and_(Order.status == 'initializing',
+                         Order.created_at + datetime.timedelta(
+                             minutes=30 + order_expiry_time) > datetime.datetime.utcnow()))).count()
```

Comment:
No, initializing = order expiry time
initialized = 30 minutes + order expiry time

Reply:
We have a initialized state too🙄